src/main/scala/com/cognite/sdk/scala/common/Resource.scala

```diff
@@ -7,8 +7,8 @@ object Resource {
   val defaultLimit: Int = 1000
 }
 
-trait BaseUri {
-  val baseUri: Uri
+trait baseUrl {
```

Comment:
Type names should be Capitalized, no?

Reply:
Yes, good catch! Forgot to make the replace all case sensitive.